test/e2e/sanity_test.go

```diff
@@ -26,3 +32,44 @@ func TestOperatorLabel(t *testing.T) {
 		t.Errorf("Expected node selector 'linux', not '%s'", osSelector)
 	}
 }
+
+func TestNoDegraded(t *testing.T) {
+	cb, err := common.NewClientBuilder("")
+	if err != nil{
+		t.Errorf("%#v", err)
+	}
+	k := cb.KubeClientOrDie("sanity-test")
+
+	kubeSharedInformer := informers.NewSharedInformerFactory(k, 20 * time.Minute)
```

Comment:
Hmm.. listers and informers are useful when controllers want events and cached responses. I think using the client to directly list seems more reasonable here.